sql/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -718,6 +718,43 @@ public void testInsertFromQueryOnDuplicateKey() throws Exception {
                "true| 2\n"));
     }
 
+    @Test
+    public void testInsertOnConflictDoNothing() {
```

Comment:
would it be possible to split this up a bit into more concrete cases?

Reply:
What do you want to break up? I think it is fine as an integration test. It tests VALUES and Subquery insert style. Breaking this up makes it harder to see the use cases of DO NOTHING.

Reply:
Right now it reads like a collection of insert statements using `DO NOTHING`, but it is not clear which scenarios of `DO NOTHING` are being tested.

It would also be fine to add some comments inline if you don't want to split it up for speed reasons. But it shouldn't be necessary for a reader of the test case to guess what exactly should be tested.
E.g. `726` and `728` is semantically the same statement. It's not clear if this is important for this test, and if so it's not clear why it is important.

Reply:
726 and 728 tests DO NOTHING with and without conflict target.

The method tests different scenarios for `DO NOTHING` to achieve more coverage. I find it quite clear and easy to read. I think this is about code style and we have a different view on it.

Reply:
Broke up the test. Hope that helps with the readability.